### DISCUSSION TITLE
Muhammad Furqan | fuee

### DISCUSSION THREAD

**Discussion by fueerqan** [onlyphantom/llm-python#26] [Practicum: Generative AI]
https://colab.research.google.com/drive/11PlBQflfqcr5mW5-x8kiI2rin0apkOwM?usp=sharing

**Comment by onlyphantom** [comment]
Well done and nice touch with adding additional prompt instructions for tabular results!

Marked as passed!

Sam

**Reply by fueerqan** [reply]
Thank you very much for the feedback Sam.
May I know what should I do next to get the certificate, etc?

Thank you
